olexandr.vakulinskiy/HW2/Task1/tamagotchi.rb

```diff
@@ -0,0 +1,111 @@
+class Tamagotchi
+
+  attr_accessor :name, :health, :hunger, :thirst, :fatigue
+
+  def initialize(name)
+    @name = name
+    @health = 10
+    @hunger = 2
+    @thirst = 2
+    @fatigue = 0
+  end
+
+  def check_health
+    if @hunger == 0 || @thirst == 0
```

Comment:
better use [predicate methods](https://github.com/rubocop-hq/ruby-style-guide#predicate-methods)
```suggestion
    if @hunger.zero? || @thirst.zero?
```